spec/spec_helper.rb

```diff
@@ -1,8 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 
-require 'coveralls'
-Coveralls.wear!('rails') do
-  add_filter("/spec/")
+if ENV["TEST_SUITE"] == "vmdb"
```

Comment:
I think we may also want to modify coverage helper as well.  Use the current pattern for "vmdb", use another pattern for "migrations", etc.